package.json

```diff
@@ -45,6 +45,7 @@
     "mock-fs": "4.10.1",
     "nock": "13.0.4",
     "semantic-release": "17.1.2",
+    "semantic-release-npm-deprecate-old-versions": "^1.0.1",
```

Comment:
[Style/Stability] We tend to defined fixed versions to prevent unexpected "silent" updates.
The idea behind it is that we should not blindly trust that all packages use semantic versioning (properly without mistakes).
And thus, sometimes a patch or minor version can sometimes introduce breaking changes.

I know this reflexion make the `^` pointless (and this feature has been implemented for a good reason) but that is the general idea we have about the fixed dependancies versions we have in our repositories.